toqito/state_props/is_abs_ppt.py

```diff
@@ -0,0 +1,87 @@
+"""Checks if a quantum state is absolutely PPT."""
+
+import numpy as np
+
+from toqito.matrix_props import is_square, is_positive_semidefinite
+from toqito.state_props import in_separable_ball, abs_ppt_constraints
+
+
+def is_abs_ppt(mat: np.ndarray, dim: int | list[int] = None) -> int:
+    r"""Determine whether or not a matrix is absolutely PPT :cite:`ref`.
+
+        This function is adapted from QETLAB.
+
+        Examples
+        ==========
+        Demonstrate how the function works with expected output.
+
+        .. jupyter-execute::
+
+            import numpy as np
+            x = np.array([[1, 2], [3, 4]])
+            print(x)
+
+        References
+        ==========
+        .. bibliography::
+            :filter: docname in docnames
+
+        :param mat: A square matrix.
+        :param dim: A 1-by-2 vector containing the dimensions of the
+                    subsystems on which :code:`mat` acts.
+        :return: :code:`1` if :code:`mat` is absolutely PPT,
+        :return: :code:`0` if :code:`mat is not absolutely PPT,
+        :return: :code:`-1` if the function could not decide.
```

Comment:
According to @vprusso, this is MATLAB convention, which is what I was referring to in an earlier comment. See https://github.com/vprusso/toqito/pull/1232#discussion_r2112387569